image/v1/types.go

```diff
@@ -605,6 +605,10 @@ type ImageBlobReferences struct {
 	// not have separate config blobs and this field will be set to nil if so.
 	// +optional
 	Config *string `json:"config" protobuf:"bytes,2,opt,name=config"`
+	// manifests is the list of manifests that compose this image. For a
+	// single architecure image, it is empty. For a multi-arch image, it
+	// consists of single architecture images.
+	Manifests []string `json:"manifests,omitempty" protobuf:"bytes,4,rep,name=manifests"`
```

Comment:
are we gonna need any validation for this?

Reply:
it's a read-only API, so I doubt we need any validation for this

Reply:
> it's a read-only API, so I doubt we need any validation for this

There should still be validation to provide guarantees about the data returned by the openshift-apiserver.  You can wire validation before the return from the REST layer.

Reply:
what if the generated response doesn't pass validation, should the apiserver panic and crash? as it'll be an indication that there is a bug